src/scripts/circleci-job-canceller/long_job_canceller.py

```diff
@@ -60,7 +79,23 @@ def find_old_workflow_ids(repo_slug, window_start, window_end, headers):
                     yield {"job_status": "too_old", "name": current_workflow['name'], "id": current_workflow['id'], "username": username}
 
 
-def main(circleapitoken, orgreposlug, output_file, commit):
+def do_we_care_about_this_pipeline(current_info, ignore, headers):
```

Comment:
I think `filter_ignored_jobs` would be a clearer name, and then you can simply use empty list checking in the `main` to decide whether to continue with a workflow.